src/rpc/misc.cpp

```diff
@@ -143,6 +143,46 @@ static UniValue createmultisig(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue getdescriptorinfo(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 1) {
+        throw std::runtime_error(
+            RPCHelpMan{"getdescriptorinfo",
+            {"\nAnalyses a descriptor.\n"},
```

Comment:
Analyze?

Reply:
@instagibbs it's funny, I stumbled across this, too, and did a little search on analyse/analyze in the code base, it's almost evenly split. So it would probably be better to do a separate PR that standardizes (standardises?) on either the British or American spelling

Reply:
I am not a fan of PRs to go around switching between different english styles, especially in comments.  All of them are valid, all can be mutually read by english speakers.  Trying to maintain consistency would just mean a never ending sequence of fixups PRs.   

We have a finite amount of resources to handle changes, they should be conserved for efforts that improve the capability or reliability of the software.

Reply:
I'm not saying it should necessarily be changed, but changing it in this PR only will not really add anything wrt consistency either (since analyze/analyse appear in more or less the same frequency in the code base). So... leave it as is?

Reply:
I had no idea it was a valid word, haha. I retract the comment!

Reply:
I thought it was a misspelling of "analysis" which made no grammatical sense.